script.js

```diff
@@ -0,0 +1,59 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const bookTitle = document.getElementById('title');
+  const bookAuthor = document.getElementById('author');
+  const article = document.getElementById('bookList');
+
+  const addButton = document.getElementById('add');
+  const saveItemToLocalStorage = (title, author) => {
+    const savedBooks = JSON.parse(localStorage.getItem('savedBooks')) || [];
+    savedBooks.push({ title, author });
+    localStorage.setItem('savedBooks', JSON.stringify(savedBooks));
+  };
+
+  const removeItemFromLocalStorage = (div) => {
+    const savedBooks = JSON.parse(localStorage.getItem('savedBooks')) || [];
+    const title = div.querySelector('p:first-child').innerHTML;
+    const author = div.querySelector('p:nth-child(2)').innerHTML;
+    const index = savedBooks.findIndex(
+      (item) => item.title === title && item.author === author,
+    );
+    if (index !== -1) {
+      savedBooks.splice(index, 1);
+      localStorage.setItem('savedBooks', JSON.stringify(savedBooks));
+    }
+  };
```

Comment:
Well, your remove function is functioning slightly as expected, allowing users to delete books. However, there are a couple of adjustments that can enhance its functionality:

- [ ] When there are two or more books with the same title and author, the current implementation sometimes deletes the wrong book from local storage. For instance, if you attempt to delete the second book, it might remove the first one's data from local storage. To address this, I recommend implementing a more precise way to identify and delete the correct book from local storage.

[Awesome-books-one.webm](https://github.com/NoelLincoln/awesome-books/assets/55840999/d92f315d-39bc-4d1d-98a7-d96e167a989e)

- [ ] [Optional] To make the removal process more accurate and efficient, I suggest adding a unique identifier (like an `id`) to each book object. This identifier should be distinct for every book. You can then use this unique `id` to precisely locate and delete the desired book from both the DOM and local storage. This approach ensures accurate book removal and maintains the integrity of your local storage data.

By incorporating these adjustments, you'll enhance the accuracy and reliability of your book removal functionality. Keep up the good work and strive for a seamless user experience! 📖🗑️🚀